app/src/main/java/com/github/wnder/MainActivity.java

```diff
@@ -1,14 +1,27 @@
 package com.github.wnder;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.appcompat.widget.Toolbar;
+import androidx.core.app.ActivityCompat;
 
+import android.Manifest;
+import android.content.Context;
 import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.location.LocationListener;
+import android.location.LocationManager;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.View;
 import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.google.android.gms.maps.model.LatLng;
```

Comment:
Since we're using MapBox wouldn't it be more logic to use the MapBox import for LatLng?

Reply:
Yes you're right, but I guess it has already be done by someone else.